### ISSUE TITLE
Publicly listed apps not appearing on email notification 

### ISSUE THREAD

**Issue by HollyJoyPhillips** [reapit/foundations#10636]
**Issue:** Two apps that have been publicly listed but not appearing on the weekly email

- 0cd32b1d-7807-4bc5-b804-47ba47c6425a - 23rd Jan
publicListedDate:"2024-01-23T10:57:43"

- a463abc2-6656-4bc7-9985-43e9dc5b2cdd - 25th Jan 
publicListedDate: "2024-01-25T12:04:40"

**Comment by plittlewood-rpt** [comment]
This is now resolved. I spotted another couple of oddities whilst looking at this. Fix includes:

Fixed bug where monthly run wouldn't calculate the end date correctly
Fixed bug where publicly listed apps were not being returned
Fixed bug where apps listed in the same month as being registered were not listed on

I've fired off this morning's as there was an issue with the job running on 1st of every month with data for the previous month not firing properly.
